bacula/baculum/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="ops@marfeel.com"
+
+RUN apt-get update && apt-get install -y --no-install-recommends gnupg wget joe ca-certificates
+RUN wget -qO - http://bacula.org/downloads/baculum/baculum.pub | apt-key add -
+
+COPY baculum.list /etc/apt/sources.list.d/baculum.list
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    gnupg \
+    wget \
+    baculum-common \
+    baculum-web \
+    baculum-web-apache2 \
+    baculum-api \
+    baculum-api-apache2
```

Comment:
Sorry, just realised that we are missing the clean steps.

```dockerfile
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
        gnupg \
        wget \
        baculum-common \
        baculum-web \
        baculum-web-apache2 \
        baculum-api \
        baculum-api-apache2 \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*
```

Source: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get